Entity/FormFieldTranslation.php

```diff
@@ -129,7 +129,7 @@ public function getField()
      */
     public function getPlaceholder()
```

Comment:
phpdoc is now wrong - should be `string|boolean` which seems to be wrong in my opinion. why not returning null if the placeholder is not set?

Reply:
there is a check for `false` and only when its really false the placeholder is not rendered.